task/ansible-tower-cli/0.1/ansible-tower-cli.yaml

```diff
@@ -38,7 +38,7 @@ spec:
       default: "tower-creds"
   steps:
     - name: config
-      image: quay.io/rcmendes/ansible-tower-cli:latest
+      image: quay.io/rcmendes/ansible-tower-cli@sha256:3a61778f410526db8d6e02e87715d58ee770c4a4faf57ac408cb5ec1a025ef2c #tag: latest
```

Comment:
would updating the `latest` tag (when the author pushes a new image) overwrite this `sha256` and cause the task to fail?

Reply:
Yes the task would fail, but for now we don't have any other tag available for it on which we can rely on

Reply:
So, it will not "fail" per-say — the digest will point to a non tagged image which still works. The main problem with that is, we do not have any guarantee that the upstream registry won't garbage collect the image (docker hub does garbage collect those untagged image at some point).